.github/workflows/farshow-build.yml

```diff
@@ -0,0 +1,98 @@
+name: Farshow build
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    container:
+      image: debian:bookworm
+
+    steps:
+    - name: Install dependencies
+      run: |
+        apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -qqy \
+          clang-format \
+          clang-tidy \
+          cmake \
+          g++ \
+          git \
+          libglew-dev \
+          libglfw3-dev \
+          libopencv-dev \
+          libopengl-dev \
+          make
+
+    - name: Add repository to safe list
```

Comment:
Why do we need that?

Reply:
The `git` command won't be initialized unless the repository directory is safe.

You can see this behavior in the "Test if changes were applied" task for the next pipeline:
https://github.com/antmicro/farshow/actions/runs/3957311195/jobs/6777523261